src/AllLayers.js

```diff
@@ -7,7 +7,7 @@ L.LayerGroup.environmentalLayers = L.LayerGroup.extend(
       currentHash: location.hash,
       // Source of Truth of Layers name .
       // please put name of Layers carefully in the the appropriate layer group.
-      layers0: ['purpleLayer', 'toxicReleaseLayer', 'pfasLayer', 'aqicnLayer', 'osmLandfillMineQuarryLayer', 'Unearthing'],
+      layers0: ['PLpeopleLayer', 'purpleLayer', 'toxicReleaseLayer', 'pfasLayer', 'aqicnLayer', 'osmLandfillMineQuarryLayer', 'Unearthing'],
```

Comment:
Hey, P should be small here, otherwise it will break.
This name should be exactly similar to Layer's function name.

Reply:
Ahh, ok.